drivers/i2c/Kconfig

```diff
@@ -267,6 +267,13 @@ config I2C_MCUX
 	help
 	  Enable the mcux I2C driver.
 
+config I2C_MCUX_LPI2C
+	bool "MCUX LPI2C driver"
```

Comment:
Does it make sense to have this be something like:

`bool "MCUX LPI2C driver (used on i.MX-RT)"`

Reply:
No, because there are some newer kinetis parts with this IP